mobile/src/api/images.queries.ts

```diff
@@ -10,7 +10,7 @@ export const useImages = () => {
       for (let index = 0; index < 50; index++) {
         try {
           const data = await axios.get(
-            'http://141.94.127.211:8000/get_unlabeled_random_event',
+            'http://57.128.107.129:8000/get_unlabeled_random_event',
```

Comment:
Maybe we can extract `http://57.128.107.129:8000` in a top level const so we don't forget to update both strings when we change the ip of the server